Library/Homebrew/cmd/install.rb

```diff
@@ -302,6 +307,98 @@ def run
         Install.perform_preinstall_checks_once
         Install.check_cc_argv(args.cc)
 
+        ask_input = lambda {
```

Comment:
Can these be dedicated methods rather than a lambda? The method this is part of is getting very long and we don't typically use `lambda` like this just for variable scoping.